src/sentry/grouping/ingest.py

```diff
@@ -167,3 +167,38 @@ def _calculate_background_grouping(
     }
     with metrics.timer("event_manager.background_grouping", tags=metric_tags):
         return calculate_event_grouping(project, event, config)
+
+
+def should_run_secondary_grouping(project: Project) -> bool:
+    result = False
+    secondary_grouping_config = project.get_option("sentry:secondary_grouping_config")
+    secondary_grouping_expiry = project.get_option("sentry:secondary_grouping_expiry")
+    if secondary_grouping_config and (secondary_grouping_expiry or 0) >= time.time():
+        result = True
+    return result
```

Comment:
Is `result` a variable for some debugging purposes? Doesn't seem needed at all.

```suggestion
    return secondary_grouping_config and (secondary_grouping_expiry or 0) >= time.time()
```

Reply:
UPDATE: Done [here](https://github.com/getsentry/sentry/pull/63728/commits/47d8142ffb7852c6e13119c7bb632dac6d889069).